Specs/blob/blob-storage-2017-07-29.json

```diff
@@ -2715,22 +2715,19 @@
         }
       ]
     },
-    "/{containerName}/{blob}?comp=lease": {
+    "/{containerName}/{blob}?comp=lease&acquire": {
       "put": {
         "tags": [
           "blob"
         ],
-        "operationId": "Blobs_Lease",
-        "description": "The Lease Blob operation establishes and manages a lock on a blob for write and delete operations",
+        "operationId": "Blobs_AcquireLease",
+        "description": "[Update] The Lease Blob operation establishes and manages a lock on a blob for write and delete operations",
         "parameters": [
           {
             "$ref": "#/parameters/Timeout"
           },
           {
-            "$ref": "#/parameters/LeaseId"
-          },
-          {
-            "$ref": "#/parameters/LeaseAction"
+            "$ref": "#/parameters/LeaseActionAcquire"
           },
           {
             "$ref": "#/parameters/LeaseBreakPeriod"
```

Comment:
It looks like x-ms-lease-break-period is only for the break operation.

Reply:
Yep, removed it on the other non Break operations, but missed it on Acquire for both container and blob.